.github/workflows/checks.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   commitlint:
     name: Lint commit messages
-    if: ${{ github.event_name == 'pull_request' }}
+    if: ${{ false }}
```

Comment:
As I already mentioned, can we enable this?

Reply:
After the strategy change, this will now rely on our commits made to this fork. Yes, we can enable it for this repository. However, we may need to disable it for our other Zendesk theme repository temporarily if our old commits don't follow the commit conventions. We can then re-enable it to enforce commit convention rules after our syncing PRs are merged with master.

Reply:
Our commits in this PR do follow the convention. We can enable it in this PR.